### DISCUSSION TITLE
Why when i add an item on repeater and reorder create an empty field

### DISCUSSION THREAD

**Discussion by FilipeDiasB** [awcodes/filament-table-repeater#124] [General]
When i reorder the repeater on drag and drop its creating a new empty field, and make bug on my code because i have sum calcs on this fields, i changed to reorderwith button and worked, but drag and drop not

![CleanShot 2024-06-25 at 12 31 59@2x](https://github.com/awcodes/filament-table-repeater/assets/100143990/d6cfc14d-16bb-4fe1-89d2-5118718edb5a)

**Comment by awcodes** [comment]
Can you provide a reproduction repo?

I can't seem to replicate the issue.

https://github.com/awcodes/filament-table-repeater/assets/3596800/7f1c1247-a7ff-4125-bacd-e49f43f37ca3
